src/v2/templates/components/curriki-eks-04-deployment.yml

```diff
@@ -184,6 +184,12 @@ Resources:
             sudo kubectl apply -f /curriki-eks/src/v2/yamls/efs-provisioner.yml
             sudo kubectl apply -f /curriki-eks/src/v2/yamls/api.yaml
             sudo kubectl apply -f /curriki-eks/src/v2/yamls/client.yaml
+            sudo kubectl apply -f /curriki-eks/src/v2/yamls/es.yml
+            sudo kubectl apply -f /curriki-eks/src/v2/yamls/es-svc.yml
+            sleep 20
```

Comment:
Why do we use sleep 20

Reply:
so elastic search gets ready and it can generate a password.